react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/PaywallFooterViewManager.kt

```diff
@@ -48,8 +49,8 @@ internal class PaywallFooterViewManager : BasePaywallViewManager<PaywallFooterVi
                 val finalWidth = maxWidth.coerceAtLeast(suggestedMinimumWidth)
                 val finalHeight = maxHeight.coerceAtLeast(suggestedMinimumHeight)
                 setMeasuredDimension(finalWidth, finalHeight)
-                (context as? ThemedReactContext)?.let { themedReactContext ->
-                    themedReactContext.runOnNativeModulesQueueThread {
+                (context.applicationContext as? ReactContext)?.let { context ->
```

Comment:
I'm not too familiar with this, but do we need to use the application context for this? Since it's only for the dimensions, I think it should be fine for now, but ideally we use the current context if possible.

Reply:
The current context is the one that's broken and that's why we need to get the application one here